warehouse/templates/manage/publishing.html

```diff
@@ -0,0 +1,156 @@
+ # limitations under the License.
+-#}
+
+{% extends "manage_project_base.html" %}
+
+{% set active_tab = 'publishing' %}
+
+{% block title %}
+  {% trans %}OpenID Connect publisher management{% endtrans %}
+{% endblock %}
+
+{% block main %}
+{% if testPyPI %}
+{% set title = "TestPyPI" %}
+{% else %}
+{% set title = "PyPI" %}
+{% endif %}
+
+{% macro provider_row(provider) -%}
+<form id="delete-provider-{{ provider.id }}" method="POST" action="{{ request.route_path('manage.project.settings.publishing', project_name=project.name) }}">
+  <input name="csrf_token" type="hidden" value="{{ request.session.get_csrf_token() }}">
+  <input name="provider_id" type="hidden" value="{{ provider.id }}">
+</form>
+
+<tr>
+  <td scope="row">
+    {{ provider.provider_name }}
+  </td>
+  <td scope="row">
+    {{ provider|string }}
+  </td>
+  <td scope="row">
+    <input form="delete-provider-{{ provider.id }}" type="submit" value="{% trans %}Remove{% endtrans %}" class="button button--danger">
+  </td>
+</tr>
+{%- endmacro %}
+
+<div class="horizontal-section">
+  <div class="site-container">
+    <h1 class="page-title">{% trans %}OpenID Connect publisher management{% endtrans %}</h1>
+
+    <p>
+      {% trans trimmed %}
+        OpenID Connect provides a flexible, credential-free mechanism for delegating
+        publishing authority for a PyPI package to a third party service,
+        like GitHub Actions.
+      {% endtrans %}
+    </p>
+
+    <p class="no-bottom-padding">
+      {% trans trimmed %}
+        PyPI projects can use trusted OpenID Connect publishers to automate their release
+        processes, without having to explicitly provision or manage API tokens.
+      {% endtrans %}
+    </p>
+
+    <h2 class="no-bottom-padding">{% trans %}Add a new provider{% endtrans %}</h2>
+    {% if oidc_enabled %}
+    <h3>GitHub</h3>
+
+    <p>
+      {% trans trimmed href="https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect" %}
+      Read more about GitHub's OpenID Connect provider <a href="{{ href }}">here</a>.
+      {% endtrans %}
+    </p>
+
+    {{ form_error_anchor(github_provider_form) }}
+    <form method="POST" action="{{ request.route_path('manage.project.settings.publishing', project_name=project.name) }}">
+      <input name="csrf_token" type="hidden" value="{{ request.session.get_csrf_token() }}">
+      {{ form_errors(github_provider_form) }}
+      <div class="form-group">
+        <label for="owner" class="form-group__label">
+          {% trans %}Owner{% endtrans %}
+          {% if github_provider_form.owner.flags.required %}
+          <span class="form-group__required">{% trans %}(required){% endtrans %}</span>
+          {% endif %}
+        </label>
+        {{ github_provider_form.owner(placeholder=gettext("owner"), autocomplete="off", autocapitalize="off", spellcheck="false", class_="form-group__field", **{"aria-describedby":"owner-errors"}) }}
```

Comment:
We could leave it this way because we do the same elsewhere, but it wouldn't hurt to use the simple form
```suggestion
        {{ github_provider_form.owner(placeholder=gettext("owner"), autocomplete="off", autocapitalize="off", spellcheck="false", class_="form-group__field", aria_describedby="owner-errors") }}
```
(see #10003 / https://wtforms.readthedocs.io/en/2.3.x/widgets/#wtforms.widgets.html_params)